catalog/task/create-github-release/0.1/create-github-release.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: create-github-release
+spec:
+  description: >-
+    Tekton task that creates a release on github.com via the GitHub API
+  params:
+    - name: repository
+      type: string
+      description: "The github repository where the release should be created"
+    - name: release_version
+      type: string
+      description: "The version string of the new release"
+    - name: githubSecret
+      type: string
+      description: "The kube secret to use to authenticate to GitHub, containing one key: token"
+    - name: content_directory
+      type: string
+      description: "The directory inside the workspace to find files for release"
+      default: "release/"
+  workspaces:
+    - name: data
+      description: The workspace where the binaries to release reside
+  results:
+    - name: url
+      type: string
+      description: URL to inspect the created release
+  steps:
+    - name: create-release-from-binaries
+      image: registry.access.redhat.com/ubi9/ubi:latest@sha256:351ed8b24d440c348486efd99587046e88bb966890a9207a5851d3a34a4dd346
+      script: |
+        #!/usr/bin/env bash
+        curl -L \
+          https://github.com/cli/cli/releases/download/v2.32.1/gh_2.32.1_linux_amd64.tar.gz \
+          --output gh_2.32.1_linux_amd64.tar.gz && \
+        tar -zvxf gh_2.32.1_linux_amd64.tar.gz && \
+        mv gh_2.32.1_linux_amd64/bin/gh /bin/gh && \
+        chmod +x /bin/gh && \
+        cd "/workspace/data/$CONTENT_DIRECTORY" && \
```

Comment:
This should be: `"$(workspaces.data.path)/$CONTENT_DIRECTORY"`

Reply:
good catch

Reply:
I copied this from the create-pyxis-image task, so it should likely be fixed there too.

Reply:
Can you link me to where you saw it in the create-pyxis-image task?

Reply:
Oh, now that I go looking for it; I can't find it. Must be my own error.

Reply:
Updated in 2cf54d5.